### PR TITLE
Add Error Messages for SpongeAuth Errors (Fixes #303)

### LINKED ISSUE #303: Failure to create an Organization doesn't give an error message
`error.spongeauth.parse` is displayed if you attempt to create an Organization with a space in the name.
`error.spongeauth.auth` is displayed when Ore can't connect to SpongeAuth.

These need to be placed in `conf/messages`

### PR DESCRIPTION
**Pull request by phase**
Everyone likes a nice error message.
Fixes #303

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -115,6 +115,9 @@ error.plugin.notSponge          =   Project is marked as a Sponge Plugin and new
 error.plugin.notForge           =   Project is marked as a Forge Mod and new version contains no dependency to Forge.
 error.plugin.fileName           =   Filename already in use. Please rename your file and try again.
 
+error.spongeauth.auth           =   Couldn't connect to SpongeAuth.
+error.spongeauth.parse          =   That name is invalid, please choose a CamelCase name like <i>ExampleOrg</i>.
```
Comment (kashike) [inline conf/messages]: Why CamelCase?

Comment (vectrixdevelops) [reply conf/messages]: Yeah, just shouldn't have spaces.

Comment (phase) [reply conf/messages]: `That name is invalid, please choose a name without spaces or symbols`?

**Comment by phase** [comment]
Updated with 
```
error.spongeauth.parse = That name is invalid, please choose a name without spaces or symbols.
```

**Pull request merged**